docs/reference/changelog.rst

```diff
@@ -0,0 +1,264 @@
+Changelog
+*********
+
+..
+  release template:
+
+  X.Y.Z (YYYY-MM-DD)
+  ------------------
+
+  Core
+  ====
+
+  # for everything related to the lifecycle of packing a charm
+
+  Bases
+  #####
+
+  <distro>@<series>
+  """""""""""""""""
+  (order from newest base to oldest base)
+
+  Plugins
+  #######
+
+  <plugin>
+  """"""""
+
+  List plugins
```

Comment:
I don't really get it, what is this "List plugins" here?

Reply:
Not really sure - I copied it wholesale from the snapcraft one. @mr-cal?

Reply:
It would be for changes to `snapcraft list-plugins` like this: https://canonical-snapcraft.readthedocs-hosted.com/en/latest/reference/changelog.html#list-plugins

If you're not using the snapcraft template, you could drop the whole comment block.  I wasn't sure if you were planning to start using it for subsequent releases.

Reply:
Ah so it's a sub-section related to the section (plugins), got it

Reply:
@mr-cal I am planning on using it for future releases :-)